### PR TITLE
Add section about template writing reference in the web UI.

### PR DESCRIPTION
**Pull request by suyogsainkar**
[RFE] Document how to find template writing reference document in the Satellite web UI
https://bugzilla.redhat.com/show_bug.cgi?id=1901937

### CODE REVIEW

**Review by ares — APPROVED** [review]

File: guides/doc-Managing_Hosts/topics/Template_Writing.adoc [hunk 1]
```diff
@@ -31,6 +31,18 @@ Note that the default templates provided by {ProjectName} (*Hosts* > *Provisioni
 When provisioning a host or running a remote job, the code in the ERB is executed and the variables are replaced with the host specific values.
 This process is referred to as *rendering*.
 The {ProjectServer} has the safemode rendering option enabled by default, which prevents any harmful code being executed from templates.
+
+
+[[appe-Red_Hat_Satellite-Managing_Hosts-Template_Writing_Reference-ui]]
+=== Template Writing Reference in the {Project} web UI
```
Comment (spetrosi) [inline guides/doc-Managing_Hosts/topics/Template_Writing.adoc:37]:
```suggestion
=== Accessing the Template Writing Reference in the {Project} web UI
```

File: guides/doc-Managing_Hosts/topics/Template_Writing.adoc [hunk 2]
```diff
@@ -31,6 +31,18 @@ Note that the default templates provided by {ProjectName} (*Hosts* > *Provisioni
 When provisioning a host or running a remote job, the code in the ERB is executed and the variables are replaced with the host specific values.
 This process is referred to as *rendering*.
 The {ProjectServer} has the safemode rendering option enabled by default, which prevents any harmful code being executed from templates.
+
+
```
Comment (spetrosi) [inline guides/doc-Managing_Hosts/topics/Template_Writing.adoc:35]: Why two blank lines?

File: guides/doc-Managing_Hosts/topics/Template_Writing.adoc [hunk 3]
```diff
@@ -31,6 +31,18 @@ Note that the default templates provided by {ProjectName} (*Hosts* > *Provisioni
...
+[[appe-Red_Hat_Satellite-Managing_Hosts-Template_Writing_Reference-ui]]
+=== Template Writing Reference in the {Project} web UI
+
+You can access the template writing reference document in the {Project} web UI. 
+
+. Log in to the {Project} web UI.
+. Navigate to *Administer* > *About*.
+. Click the *Templates DSL* link in the Support section.
+ 
+
```
Comment (spetrosi) [inline guides/doc-Managing_Hosts/topics/Template_Writing.adoc:45]: Why two blank lines?

**Review by spetrosi — CHANGES_REQUESTED** [review]

File: guides/doc-Managing_Hosts/topics/Template_Writing.adoc (hunk 1 above)
Comment (suyogsainkar) [reply guides/doc-Managing_Hosts/topics/Template_Writing.adoc]: Done

File: guides/doc-Managing_Hosts/topics/Template_Writing.adoc (hunk 2 above)
Comment (suyogsainkar) [reply guides/doc-Managing_Hosts/topics/Template_Writing.adoc]: Fixed

File: guides/doc-Managing_Hosts/topics/Template_Writing.adoc (hunk 3 above)
Comment (suyogsainkar) [reply guides/doc-Managing_Hosts/topics/Template_Writing.adoc:43]: Fixed

**Comment by suyogsainkar** [comment]
@spetrosi Thanks for the review. I've applied those changes.

**Pull request merged**